validations/src/validations.rs

```diff
@@ -2733,89 +2733,89 @@ mod tests {
             .unwrap();
 
         let (_, p00) = calculate_reppoe_threshold(&rep_engine, &ids[0], 1);
```

Comment:
Maybe we could simplify this test by adding a helper function that calculates the threshold of all the identities in order, like

```rust
let rep_thresholds = |thres| {
    vec![
        (calculate_reppoe_threshold(&rep_engine, &ids[0], thres).1 * 100_f64).round() as u32,
        (calculate_reppoe_threshold(&rep_engine, &ids[1], thres).1 * 100_f64).round() as u32,
        // TODO: use a for loop
    ]
};

let expected_rep_1 = vec![28, 24, 18, 9, 5, 14, 1, 1];
assert_eq!(rep_thresholds(1), expected_rep_1);
```